app/src/main/java/ar/com/wolox/android/example/ui/home/HomeFragment.kt

```diff
@@ -2,11 +2,53 @@ package ar.com.wolox.android.example.ui.home
 
 import ar.com.wolox.android.R
 import ar.com.wolox.android.databinding.FragmentHomeBinding
+import ar.com.wolox.android.example.ui.home.news.NewsFragment
+import ar.com.wolox.android.example.ui.home.profile.ProfileFragment
 import ar.com.wolox.android.example.ui.login.LoginActivity
+import ar.com.wolox.android.example.utils.enableTabColorChange
+import ar.com.wolox.wolmo.core.adapter.viewpager.SimpleFragmentPagerAdapter
 import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import javax.inject.Inject
 
 class HomeFragment : WolmoFragment<FragmentHomeBinding, HomePresenter>(), HomeView {
-    override fun init() {}
+
+    @Inject
+    internal lateinit var newsFragment: NewsFragment
+
+    @Inject
+    internal lateinit var profileFragment: ProfileFragment
+
+    override fun init() {
+        with(binding) {
+            viewPager.adapter = SimpleFragmentPagerAdapter(childFragmentManager).apply {
+                addFragments(
+                        newsFragment to "Page 1",
```

Comment:
The keys of each fragment should be a string that represent each fragment, not page 1 and page 2. Both strings could be the class name or a constant declared in companion object